src/response.rs

```diff
@@ -1,11 +1,15 @@
 use http_service::Body;
 
-use crate::body;
-
-/// An HTTP response.
-///
-/// A convenient alias for the `http::Response` type, using Tide's `Body`.
-pub type Response = http::Response<Body>;
+pub type Response = http_service::Response;
+
+pub fn json<T: serde::Serialize>(t: T) -> Response {
```

Comment:
Perhaps return a `Result` to be more rust idiomatic? https://github.com/rustasync/tide/pull/156#issuecomment-480596914